### PR TITLE
[Integration][Snyk, PD, Argo, NewRelic, ServiceNow] Add target=blank to integrations

### PR DESCRIPTION
**Pull request by PeyGis**
# Description

What - Added target=blank attribute to all html links in the spec.yaml for Snyk, SonarQube, ArgoCD, NewRelic, ServiceNow and PagerDuty
Why -
How -

## Type of change

Please leave one option from the following and delete the rest:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] New Integration (non-breaking change which adds a new integration)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Non-breaking change (fix of existing functionality that will not change current behavior)
- [ ] Documentation (added/updated documentation)

## Screenshots

Include screenshots from your environment showing how the resources of the integration will look.

## API Documentation

Provide links to the API documentation used for this integration.

### CODE REVIEW

File: integrations/argocd/CHANGELOG.md [hunk 1]
```diff
@@ -11,7 +11,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Improvements
```
Comment (matan84) [inline integrations/argocd/CHANGELOG.md:13]: Create new version for each integration

**Review by matan84 — CHANGES_REQUESTED** [review]
Nice, left one comment

File: integrations/newrelic/.port/spec.yaml [hunk 2]
```diff
@@ -12,14 +12,14 @@ configurations:
...
   - name: newRelicGraphqlURL
     required: false
     type: string
     default: https://api.newrelic.com/graphql
-    description: URL for New Relic GraphQL API. If not provided, the default value will be https://api.newrelic.com/graphql. Users in the EU region should use https://api.eu.newrelic.com/graphql. For more information on GraphQL URL, see the <a href="https://docs.newrelic.com/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph/"> New Relic documentation</a>.
+    description: URL for New Relic GraphQL API. If not provided, the default value will be https://api.newrelic.com/graphql. Users in the EU region should use https://api.eu.newrelic.com/graphql. For more information on GraphQL URL, see the <a href="https://docs.newrelic.com/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph/" target="_blank"> New Relic documentation</a>.
```
Comment (Tankilevitch) [inline integrations/newrelic/.port/spec.yaml:25]:
```suggestion
    description: URL for New Relic GraphQL API. If not provided, the default value will be https://api.newrelic.com/graphql. Users in the EU region should use <a href="https://api.eu.newrelic.com/graphql" target="_blank"> https://api.eu.newrelic.com/graphql</a>. For more information on GraphQL URL, see the <a href="https://docs.newrelic.com/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph/" target="_blank"> New Relic documentation</a>.
```

File: integrations/sonarqube/.port/spec.yaml [hunk 3]
```diff
@@ -15,22 +15,22 @@ configurations:
...
   - name: appHost
     type: string
     required: false
-    description: The host to subscribe webhooks to
+    description: The host of the Port Ocean app. Used to set up the integration endpoint as the target for webhooks created in SonarQube
   - name: sonarUrl
     type: string
     required: false
-    description: The SonarQube URL
+    description: The SonarQube URL to connect to. This is required if you are using SonarQube on-premise. If you are using SonarCloud, the default URL is https://sonarcloud.io
```
Comment (Tankilevitch) [inline integrations/sonarqube/.port/spec.yaml:30]:
```suggestion
    description: The SonarQube URL to connect to. This is required if you are using SonarQube on-premise. If you are using SonarCloud, the default URL is <a href="https://sonarcloud.io" target="_blank">https://sonarcloud.io</a>
```

**Review by Tankilevitch — APPROVED** [review]

**Pull request merged**